hack/util.sh

```diff
@@ -459,9 +459,7 @@ function install_registry() {
 readonly -f install_registry
 
 function wait_for_registry() {
-	local generation="$(oc get dc/docker-registry -o 'jsonpath={.metadata.generation}')"
-	local onereplicajs='{.status.observedGeneration},{.status.replicas},{.status.updatedReplicas},{.status.availableReplicas}'
-	wait_for_command "oc get dc/docker-registry -o 'jsonpath=${onereplicajs}' --config='${ADMIN_KUBECONFIG}' | grep '^${generation},1,1,1$'"  "$((5*TIME_MIN))"
+	wait_for_command "oc rollout status dc/docker-registry --request-timeout=1m"  "$((5*TIME_MIN))"
```

Comment:
As @liggitt mentioned, one of the core functional bits of this method _is_ the fact that it's pegging to a specific generation and waiting for that generation to be ready. `oc rollout status` may be appropriate in the initial deploy & wait for readiness stage, but not in the update stages.